index.html

```diff
@@ -1063,29 +1059,29 @@ <h3>
               </tr>
               <tr>
                 <th>
-                  <a href="https://lists.w3.org/Archives/Member/w3c-ac-members/2024AprJun/0026.html">Charter Extension</a>
+                  Working on new charter
                 </th>
                 <td>
-                  21 April 2024
+                  27 March 2024
```

Comment:
```suggestion
                  2024-03-27
```

Reply:
@TallTed thanks. I will take these as a bulk update, but not all.  There are sections (e.g., Communications) which are verbatim copies of the official charter template, and I prefer to keep them this way. It may lead to unnecessary discussions if we introduce a change, even if it is grammatical only...